sagemaker-pipelines/tabular/caching/sagemaker-pipelines-caching-steps.ipynb

```diff
@@ -0,0 +1,936 @@
+{
```

Comment:
<blockquote>Before input code files from the local file system are uploaded to S3, they are hashed, and the hash value is included in the S3 path</blockquote><p>Can we add more words on how this improve the pipeline cache, given that this is mainly examined in the next part of this notebook.</p><p>For example,</p><ul><li>Are the file contents are hashed?</li><li>If the file content is not changed and no other critical arguments changes, we should expect a cache hit?</li><li>If the file content is changed we would expect a cache miss?</li></ul><p> </p><p>Or we can point out here that this would be examined in the next part of the notebook and add more explanations for the cache hit and miss cases in the next part of the notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>